packages/synapse-react-client/src/components/AccessRequirementList/AccessRequirementList.tsx

```diff
@@ -46,17 +41,21 @@ import TwoFactorAuthEnabledRequirement from './RequirementItem/TwoFactorAuthEnab
 import { AccessRequirementListItem } from './AccessRequirementListItem'
 import { useSynapseContext } from '../../utils/context/SynapseContext'
 import { useCanShowManagedACTWikiInWizard } from './AccessRequirementListUtils'
-import { DialogActions } from '@mui/material'
+import { noop } from 'lodash-es'
+import { DialogBase } from '../DialogBase'
 
 export type AccessRequirementListProps = {
   entityId: string // will show this entity info
   teamId?: string // will show this team info
   accessRequirementFromProps?: Array<AccessRequirement>
-  onHide: () => void
-  renderAsModal?: boolean
   numberOfFilesAffected?: number // if provided, will show this instead of the entity information
   requestObjectName?: string // if provided, will show this instead of the entity information or numberOfFilesAffected
+  /* If true, this component will render in its own modal */
+  renderAsModal?: boolean
+  /* Overrides the default dialog title. This only applies if renderAsModal is true */
   dialogTitle?: string // if provided, will show this instead of Data Access Request
+  /* Called when the modal is hidden. This only applies if renderAsModal is true */
+  onHide?: () => void
 }
```

Comment:
Don't mind the inconsistent comment placement for now, it will be made consistent in a follow-up PR that will make further changes this component